common/config/rush/command-line.json

```diff
@@ -57,6 +57,14 @@
       "shellCommand": "rush update --variant stable",
       "safeForSimultaneousRushProcesses": true
     },
+    {
+      "commandKind": "global",
+      "name": "update:full",
+      "summary": "Update dependencies for all build flavours to latest SemVers.",
```

Comment:
When would this command be used?

Reply:
`rush` recommends running this "regularly".

Azure SDK for JS actually has automation to run this weekly: https://github.com/Azure/azure-sdk-for-js/blob/1af9322b9a13e7b3dd758e30d55c2cff71f9fb03/eng/pipelines/weekly_automation.yml#L5